server/sqlstore/playbook.go

```diff
@@ -352,7 +363,8 @@ func (p *playbookStore) GetPlaybooksForTeam(requesterInfo app.RequesterInfo, tea
 				CASE WHEN p.WebhookOnCreationEnabled THEN 1 ELSE 0 END +
 				CASE WHEN p.MessageOnJoinEnabled THEN 1 ELSE 0 END +
 				CASE WHEN p.WebhookOnStatusUpdateEnabled THEN 1 ELSE 0 END +
-				CASE WHEN p.SignalAnyKeywordsEnabled THEN 1 ELSE 0 END
+				CASE WHEN p.SignalAnyKeywordsEnabled THEN 1 ELSE 0 END +
+				CASE WHEN p.CategorizeChannelEnabled THEN 1 ELSE 0 END
```

Comment:
`ExportChannelOnArchiveEnabled`
```suggestion
				CASE WHEN p.CategorizeChannelEnabled THEN 1 ELSE 0 END + 
				CASE WHEN ExportChannelOnArchiveEnabled THEN 1 ELSE 0 END
```